packages/slate-cli/src/utils.js

```diff
@@ -178,3 +197,13 @@ export function isShopifyThemeWhitelistedDir(directory) {
   const whitelist = ['assets', 'layout', 'config', 'locales', 'sections', 'snippets', 'templates'];
   return whitelist.indexOf(directory) > -1;
 }
+
+/**
+ * Tests if file is a Shopify theme settings file (settings_*.json)
+ *
+ * @param {string} directory - The path to the directory.
+ */
+export function isShopifyThemeSettingsFile(file) {
+  const whitelist = ['settings_schema.json', 'settings_data.json'];
```

Comment:
These values are buried pretty deep in the code. Makes me think this package could use a config file. Definitely not needed in this PR though.